_posts/2018-11-30-Rates-Rebate.md

```diff
@@ -0,0 +1,453 @@
+---
+title: Rates Rebates (Te Whakamāmā i ngā Reiti)
+layout: report
+---
+![RR logo](/assets/media/Rates-Rebates/RR1.png)
+
+__Authored by:__ Siobhan McCarthy and Glen Thurston
+
+# Contents
+1. [Overview](#Go-to-Overview)
+2. [Background](#Go-to-Background)
+3. [The Discovery](#Go-to-The-Discovery)
+4. [The Alpha](#Go-to-The-Alpha)
+5. [Insights from Discovery and Alpha](#Go-to-Insights-Discovery-Alpha)
+6. [Next Steps](#Go-to-Next-Steps)
+7. [Appendix One - Service Blueprints](#Go-to-Appendix1)
+8. [Appendix Two - Trend Analysis](#Go-to-Appendix2)
+9. [Appendix Three - Comparison between the web app & current paper application](#Go-to-Appendix3)
+
+A collaborative initiative between Department of Internal Affairs and Local Authorities supported by the Service Innovation Lab 
+
+## Overview<a name = "Go-to-Overview"></a>
+
+[' A Rates Rebates’](https://www.govt.nz/browse/housing-and-property/getting-help-with-housing/getting-a-rates-rebate/#who-can-apply) is a subsidy (up to $630) for low-income homeowners on the cost of their residential rates. The subsidy is delivered by Local Councils and is administered by Central Government (specifically the Department of Internal Affairs - DIA).
```

Comment:
@JacOng17  I think Brenda was referring to ```[' A Rates Rebates’]```. There is a space before A and `’` after Rebates. Should be `['A Rates Rebates']`.
Also you might want to double check it should be ['A Rates Rebate'] or ['A Rates Rebates'].

Reply:
Have fixed that already :smile:  but according to the report, it's 'A Rates Rebates'
![from the report](https://user-images.githubusercontent.com/13788498/49317904-3da73f00-f55b-11e8-9aba-81f1a54e55cd.png)